configure.ac

```
@@ -301,9 +300,11 @@ except Exception:
    sys.exit(1)
 EOF
   AS_IF([test $? -eq 0],
-    [have_requests=yes],
-    [have_requests=no])
+    [REQUESTS=$PYTHON; have_requests=yes],
+    [REQUEST=;         have_requests=no])
```

Should these variables be the same?
```suggestion
    [REQUESTS=$PYTHON; have_requests=yes],
    [REQUESTS=;         have_requests=no])
```